name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RenaultBankDirektPDFExtractor.java

```diff
@@ -16,18 +13,19 @@
 @SuppressWarnings("nls")
 public class RenaultBankDirektPDFExtractor extends AbstractPDFExtractor
 {
-    private static final String DEPOSIT_2019 = "^(?<date>\\d+.\\d+.)  (.*-Gutschrift)(\\s*)(?<amount>[\\d\\s,.]*)(\\+$)";
-    private static final String REMOVAL_2019 = "^(?<date>\\d+.\\d+.)  (Internet-Euro-Überweisung)(\\s*)(?<amount>[\\d\\s,.]*)(\\-)(.*)";
-    private static final String INTEREST_2019 = "^(?<date>\\d+.\\d+.)  (.*)(Zinsen\\/Kontoführung)(\\s*)(?<amount>[\\d\\s,.]*)(\\+$)";
+    private static final String DEPOSIT_2019 = "^(?<date>\\d+.\\d+.)  (.*-Gutschrift)(\\s*)(?<amount>[\\d\\s,.]*)[+](\\s*)$";
+    private static final String REMOVAL_2019 = "^(?<date>\\d+.\\d+.)  (Wertstellung: \\d+.\\d+. )?(Internet-Euro-Überweisung)(\\s*)(?<amount>[\\d\\s,.]*)[-](\\s*)$";
```

Comment:
the same like the https://github.com/portfolio-performance/portfolio/pull/3775... with comment https://github.com/portfolio-performance/portfolio/pull/3775#discussion_r1481827304

The rest look quite good!! 💯

Reply:
ok, changed.